libfwupdplugin/fu-drm-device.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2023 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#define G_LOG_DOMAIN "FuDrmDevice"
+
+#include "config.h"
+
+#include "fu-bytes.h"
+#include "fu-drm-device.h"
+#include "fu-string.h"
+
+/**
+ * FuDrmDevice
+ *
+ * A DRM device.
+ *
+ * See also: #FuUdevDevice
+ */
+
+G_DEFINE_TYPE(FuDrmDevice, fu_drm_device, FU_TYPE_UDEV_DEVICE)
+
+static void
+fu_drm_device_to_string(FuDevice *device, guint idt, GString *str)
+{
+	FuDrmDevice *self = FU_DRM_DEVICE(device);
+	fu_string_append(str,
+			 idt,
+			 "Status",
+			 fu_display_state_to_string(fu_drm_device_get_state(self)));
+}
+
+/**
+ * fu_drm_device_get_state:
+ * @self: a #FuDrmDevice
+ *
+ * Gets the current status of the DRM device.
+ *
+ * Returns: a #FuDisplayState, e.g. %FU_DISPLAY_STATE_CONNECTED
+ *
+ * Since: 1.9.7
+ **/
+FuDisplayState
+fu_drm_device_get_state(FuDrmDevice *self)
+{
+	const gchar *status;
+	g_return_val_if_fail(FU_IS_DRM_DEVICE(self), G_MAXUINT);
```

Comment:
shouldn't this be `FU_DISPLAY_STATE_UNKNOWN`

Reply:
Yup, indeed, thanks.